jcl/source/common/JclSysInfo.pas

```diff
@@ -2619,8 +2621,16 @@ function GetBIOSDate: TDateTime;
   RegSeparator: Char;
   {$ENDIF ~RTL150_UP}
 begin
-  if IsWinNT then
-    RegStr := RegReadString(HKEY_LOCAL_MACHINE, WinNT_REG_PATH, WinNT_REG_KEY)
+  if IsWinNT then begin
+    {fs 14/12/2022: location of the Bios date seems to have changed on newer systems (From windows 10 ?)
+    The new location seems to exist since a while, but older location disappeared on newer OS}
```

Comment:
No need for the initials and the date. Also, two lines comments like this one should not use the curly braces style but rather the two slashes style, repeated on both lines